alfresco-transform-imagemagick/alfresco-transform-imagemagick-boot/pom.xml

```diff
@@ -81,19 +81,23 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
-                <configuration>
```

Comment:
Is this getting removed intentionally?

Reply:
Yes this is now moved to the super pom so that it can be accessed by both core-aio and imagemagick. This configuration is only used for test purposes and is not used in the final JARs.